packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -419,6 +420,22 @@ sdk.server('https://eu.api.example.com/v14');`),
     return sourceFile;
   }
 
+  /**
+   * Creates a `.gitignore` file to prevent the `dist/` directory from being tracked.
+   *
+   */
+  createGitIgnore() {
+    const file = `# This file prevents the \`dist/\` directory from being tracked via git.
```

Comment:
how do you feel about pulling this and the README file template out into isolated files that we do an `fs.readFileSync` on here? maybe put them in `src/codegen/languages/templates`?

Reply:
the gitignore feels TS-specific, right? and i was actually thinking of adding some docs specific to TS in the README as well

Reply:
ah yeah that's fair. want to at least pull them out into a TS-specific template directory? might be easier having them as flatfiles so you don't have to escape backticks.

Reply:
happy to do that with the gitignore file! but honestly since we use a [decent amount of templated data](https://github.com/readmeio/api/blob/08925bc4c262ed9d3a4492b7e6e385d5276a499c/packages/api/src/codegen/languages/typescript/index.ts#L665-L667) in the README file i don't think we gain much by moving it to a separate file and then setting up find/replace logic

Reply:
k that's cool to leave it alone